### ISSUE TITLE
Move the region down?

### ISSUE THREAD

**Issue by bobrocke** [atom/bracket-matcher#135]
Is it possible to offset the `.bracket-matcher .region` down? I'd like the underline to be a little lower than it is now so as not to overlap the bottom of `}` and `[`s.

**Comment by mnquintana** [comment]
@bobrocke What font are you using (and font size)? That's definitely not the case with all fonts. (I'm using Fira Mono, and the `.bracket-matcher .region` isn't touching the bottom of `{`s or `[`s)

**Comment by bobrocke** [comment]
Source Code Pro at 12px.

Bob
-----
Bob Rockefeller
Richmond Hill, GA
www.bobrockefeller.com

**Comment by bobrocke** [comment]
Yes, with Fira Mono, there is a space between the underline and the symbols. But I like Source Code Pro better and it seems many others do, too. Is there a way to accommodate both?

**Comment by lee-dohm** [comment]
Maybe add `padding-bottom: 1px` or so to the `.bracket-matcher .region`?

**Comment by bobrocke** [comment]
`padding-bottom` appears to have no effect. I've switched fonts to Menlo, which works like Fira Mono in this regard. But I'd still like to figure this out and go back to Source Code Pro.
